core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -131,6 +136,32 @@ class TestSuiteWriter {
         return lines.toString()
     }
 
+    private fun handleResetDatabaseInput(solution: Solution<*>): String{
+        if (!config.outputFormat.isJavaOrKotlin())
+            throw IllegalStateException("DO NOT SUPPORT resetDatabased for "+ config.outputFormat)
+
+        val accessedTable = mutableSetOf<String>()
+        solution.individuals.forEach { e->
+            if (e.individual is ApiWsIndividual){
```

Comment:
add `//TODO will need to be refactored when supporting Web Frontend`

Reply:
thanks, added